components/raftstore/src/store/async_io/write.rs

```diff
@@ -584,13 +592,17 @@ where
         let send_time = duration_to_sec(now2.saturating_duration_since(now));
         STORE_WRITE_SEND_DURATION_HISTOGRAM.observe(send_time);
 
-        for (region_id, (peer_id, ready_number)) in &self.batch.readies {
-            self.notifier
-                .notify_persisted(*region_id, *peer_id, *ready_number);
+        let mut callback_time = 0f64;
```

Comment:
Seems can be moved into `if notify` block

Reply:
The `slow_log` below needs this `callback_time`.